python/ray/experimental/state/common.py

```diff
@@ -516,6 +547,35 @@ class ClusterEventState(StateSchema):
     custom_fields: Optional[dict] = state_column(filterable=False, detail=True)
 
 
+class Humanify:
+    """A class containing default methods to convert units into a human readable string."""
+
+    def timestamp(x: float):
+        """Converts miliseconds to a datetime object."""
+        return str(datetime.datetime.fromtimestamp(x / 1000))
+
+    def memory(x: int):
```

Comment:
nice

Reply:
btw, thanks for reviewing! I got 20/20 for my school open source requirement assessment :)